src/libtexture/imagecache.cpp

```diff
@@ -3551,6 +3551,8 @@ ImageCacheFile::udim_setup()
                     v -= 1;
                 }
             }
+            if (u < 0 || v < 0)
+                continue;  // invalid tile
```

Comment:
It would probably pay to spit out a warning here to the user to tell them.
Allowing the process to continue, un-notified, could be rather perplexing if the user is unaware of the issue, and the renderer also doesn't know what just happened.